comp/core/bundle.go

```diff
@@ -36,4 +37,5 @@ var Bundle = fxutil.Bundle(
 	sysprobeconfigimpl.Module,
 	telemetry.Module,
 	hostnameimpl.Module,
+	statsd.Module,
```

Comment:
Are we sure we want this as part of the core bundle? 
Core is usually reserved for components that are required by all agent flavors and subcommands. I don't think statsd is used that widely? (Please correct me if I am wrong)

Reply:
I think all agents should be sending at least a `running` metric no ? If `telemetry` is core then clearly `statsd` should be

Reply:
I think the core agent submits it's running metric directly to the aggregator. I am also not aware of the core agent sending metrics to itself over the dogstastd protocol. So that is one case that comes to mind. 

But I also see your point, I don't think it's explicitly hurting anything being here. I think @ogaca-dd had similar feedback, curious as to what his opinion is.

Reply:
I don't think this should be part of the core bundle at the moment, we can pull it into the core bundle if need be in the future.